nbconvert/preprocessors/csshtmlheader.py

```diff
@@ -66,16 +70,69 @@ def _generate_header(self, resources):
         # Construct path to Jupyter CSS
         sheet_filename = os.path.join(
             os.path.dirname(nbconvert.resources.__file__),
-            'style.min.css',
+            'index.css',
         )
         
         # Load style CSS file.
         with io.open(sheet_filename, encoding='utf-8') as f:
             header.append(f.read())
 
+        # Construct path to theme CSS
+        theme_filename = os.path.join(
+            os.path.dirname(nbconvert.resources.__file__),
+            'theme-%s.css' % self.theme,
+        )
+        
+        # Load style theme file.
+        with io.open(theme_filename, encoding='utf-8') as f:
+            header.append(f.read())
+
         # Add pygments CSS
-        formatter = HtmlFormatter(style=self.style)
-        pygments_css = formatter.get_style_defs(self.highlight_class)
+        # formatter = HtmlFormatter(style=self.style)
```

Comment:
I know you are working on allowing pygments styles to include css, but I just want to emphasize that this should be changed back before merging because this breaks customizing highlighting style. Also, I don't think we can make the new style the default style without bumping our minimum supported pygments version (assuming that your PR over at pygments gets merged). If pygments styles can be made to include css, then it would be nice to have a style for the classic notebook too. If we separate the lab and classic notebook styles into different templates, we could write some logic to use the respective styles by default. (and checking if ```self.style=='default'``` to still allow overriding).

Reply:
Don't worry. I doubt this becomes the default template right away - and the intent is very much to keep the ability to select a pygments theme.

Reply:
Quick update: the pygments [pull request](https://bitbucket.org/birkenfeld/pygments-main/pull-requests/814/allow-for-css-variable-in-pygments/) just got in.

Unfortunately it did not make it for pygments 2.4 but should be in 2.4.1 in a couple of weeks.

Reply:
@t-makaro pygments 2.4.1 was released including my patch.

This PR now makes use of the `style` attribute again, which can be a string (as before) or a type inheriting from `pygments.style.Style`.

Reply:
Resolving that part of the review.